portal/src/components/dropperV2/DropperV2NewDrop.tsx

```diff
@@ -111,47 +116,49 @@ const DropperV2NewDrop = ({ address, onClose }: { address: string; onClose: () =
 
   const dropTypesIds = [20, 721, 1155, 1];
 
-  const isTokenTypeValid = dropTypesIds.includes(tokenType);
-  const isAmountValid = !isNaN(Number(tokenId));
-  const isAuthorizationTokenAddressValid = web3.utils.isAddress(authorizationTokenAddress);
-  const isTokenIdValid = !isNaN(Number(tokenId));
-  const isAuthorizationPoolIdValid = !isNaN(Number(tokenId)); //TODO Check with contract ?
-  const isTokenAddressValid = web3.utils.isAddress(tokenAddress);
-  const newMetadata = useLink({ link: uri });
+  const isTokenTypeValid = dropTypesIds.includes(state.tokenType);
+  const isAmountValid = !isNaN(Number(state.tokenId));
+  const isAuthorizationTokenAddressValid = web3.utils.isAddress(state.authorizationTokenAddress);
+  const isTokenIdValid = !isNaN(Number(state.tokenId));
```

Comment:
Can we disable the tokenId field for erc20 and erc721 drops and default the tokenId to 0?

Reply:
Force tokenId to be non-negative and integer?